src/cart/crt_rpc.c

```diff
@@ -9,8 +9,15 @@
 #define D_LOGFAC	DD_FAC(rpc)
 
 #include <semaphore.h>
-
+#include <daos_task.h>
 #include "crt_internal.h"
+#include <daos/rpc.h>
+
+#include "../mgmt/rpc.h"
+#include "../pool/rpc.h"
+#include "../container/rpc.h"
+#include "../pipeline/pipeline_rpc.h"
+#include "../object/obj_rpc.h"
```

Comment:
those includes are not particularly clean but I understand why you did that...

Reply:
Yeah, not a big fan of it myself. 
Eventually it would be good to reorganize some of these headers too, separating rpc-specific stuff from everything else and bringing all rpc definitions to the same form. Some modules today provide opc-to-str functions, some dont, some have rpc list, others have client and server lists and so on. Some headers like rdb's rpc.h couldnt be included at all as they start pulling too much extra stuff

Reply:
if you can I think it would be good to add a comment about it, as a TODO item + explanation of limitations (in case someone else wants to pick that up).

Reply:
As discussed in cart meeting these includes will be gone hopefully in the subsequent patch, while we already have 2 TODOs in the code itself indicating modules that need to be added.